tests/rt_gfsv17opn.conf

```diff
@@ -0,0 +1,2 @@
+COMPILE | s2sw_32bit_pdlib  | intel | -DAPP=S2SW -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON| - noaacloud | fv3 |
```

Comment:
We need to fix this file; I renamed the "conf" file as just ``rt.gfsv17opn``, by mistake at some point. 

We need to rename the rt.gfsv17opn to rt_gfsv17opn.conf. 

Sorry for the confusion.